pages/index.js

```diff
@@ -30,12 +32,12 @@ export default function Home({ allPostsData }) {
         <ul className={utilStyles.list}>
           {allPostsData.map(({ id, date, title }) => (
             <li className={utilStyles.listItem} key={id}>
-              {title}
-              <br />
-              {id}
-              <br />
-              {date}
-            </li>
+            <Link href={`/posts/${id}`}>{title}</Link>
+            <br />
+            <small className={utilStyles.lightText}>
```

Comment:
esto no es asi en verdad es asá, que penca el qlo

```suggestion
            <small className={utilStyles.lightText}>
            omg
```

Reply:
ok